_posts/2015-01-08-thank-you-open-source-community.md

```diff
@@ -0,0 +1,65 @@
+* [AFNetworking](https://github.com/AFNetworking/AFNetworking) is great for all things networking, but we especially like `UIImageView+AFNetworking` for async image loading.
+* [Alcatraz](https://github.com/supermarin/Alcatraz) is the package manager that brings XVim to Xcode, and that makes me happy.
+* [BZGFormViewController](https://github.com/benzguo/BZGFormViewController) is great for simple forms that require validation. We use it in for in our sign up and edit profile views.
+* [BZGMailgunEmailValidation](https://github.com/benzguo/BZGMailgunEmailValidation) is perfect if you use Mailgun for email validation.
+* [BlocksKit](https://github.com/zwaldowski/BlocksKit) because `dismissWithClickedButtonIndex:animated:` delegate methods are no fun at all. Besides, who “clicks” on an iOS device?
+* [Braintree](https://github.com/braintree/braintree_ios) allows our users to pay via Venmo on apps like [YPlan](https://yplanapp.com/).
+* [CMPopTipView](https://github.com/chrismiles/CMPopTipView) has been useful when we want to draw attention to a new feature that we added.
+* [CocoaPods](https://github.com/cocoapods/cocoapods) is how we manage all of our dependencies, including private pods. Can’t wait for their 0.36 release which supports Swift!
+* [DAZABTest](https://github.com/dasmer/DAZABTest) provides a super simple API to do basic A/B tests.
+* [Expecta](https://github.com/specta/expecta) is a great matcher framework that makes your tests read like English. `expect(myTests).toNot.beEmpty()`
+* [FLEX](https://github.com/Flipboard/FLEX) is built into all of our debug and dogfood builds, and it makes finding and fixing UI bugs so much fun and so much easier.
+* [FXBlurView](https://github.com/nicklockwood/FXBlurView) makes it super easy to have your own blurred views to match iOS 7 and 8’s frosty look.
+* [FrameAccessor](https://github.com/AlexDenisov/FrameAccessor) is perfect for the lazy programmer who would prefer to type `view.width = 50` intead of `CGRect newFrame = view.frame; newFrame.size.width = 50; view.frame = newFrame;`
+* [GBDeviceInfo](https://github.com/lmirosevic/GBDeviceInfo) tells us useful things about the device our app is running on so we can optimize a little for older devices, etc.
+* [Godzippa](https://github.com/mattt/Godzippa) has been immensely helpful when uploading large amounts of data to our API.
+* [iRate](https://github.com/nicklockwood/iRate) because we love hearing back from our users. <3
+* [JTSImageViewController](https://github.com/jaredsinclair/JTSImageViewController) is what you see when you tap on a picture on a friend’s profile. We love the interaction where you can flick the image off the screen.
+* [KGStatusBar](https://github.com/kevingibbon/KGStatusBar) We use this to show “offline mode” in the status bar for Braintree merchants to test out Venmo integration. Super simple to use!
```

Comment:
:memo: tiny nitpick, but this is the only one that starts with a complete sentence. Maybe change to something like `KGStatusBar is used to` :)